rule_schema_v1.yaml

```diff
@@ -759,6 +765,9 @@ properties:
         match:
           $ref: "#/$defs/new-pattern"
         # EXPERIMENTAL
+        metavars:
```

Comment:
it's at the same level than a match:?

Reply:
yes, it may occur at the outermost (if the outermost pattern is a `pattern-regex`)

like:
```
rules:
  - id: foo
    languages: [python]
    severity: ERROR
    message: "FOO"
    pattern-regex: "(.*)"
    metavars:
      - 1: $A
```